live_pipeline/extract_games.py

```diff
@@ -0,0 +1,122 @@
+from urllib.request import urlopen
+from bs4 import BeautifulSoup
+import pandas as pd
+import requests
+
+
+def get_html(url):
+    """function to open the url and get the information"""
+    page = urlopen(url)
+    html_bytes = page.read()
+    html = html_bytes.decode("utf_8")
+    return html
```

Comment:
We could implement some error handling here using a try-except block to ensure we don't crash the code if there is something wrong with opening the url? E.g. 

`def get_html(url):
    try:
        response = requests.get(url)
        response.raise_for_status()
        html = response.text
        return html
    except Exception as e:
        print(f"Error: {e}")
        return None`
  
If you can implement a more specific exception type for the error raised even better!